### ISSUE TITLE
Add host nothing happen after Ip Address

### ISSUE THREAD

**Issue by knives49** [wackerl91/script.luna#117]
- Luna Version:0.7.0 RC3
- nVidia GeForce Experience Version: 3.2.0.96
- Moonlight Embedded Version:2.2.3
- OSMC Version: the last with all update

**Expected Behaviour:**
Show the added host
**Actual Behaviour:**
when I add host, I have the pop-up for enter IP address but nothing happen after.

The log.
http://pastebin.com/WMSxgFnH
the first time, with aeon nox skin, the second time with the classic skin.

**Comment by wackerl91** [comment]
Tested the fix in the following way (there's no need to do this on your end, just want to explain what happened):
- uninstalled zeroconf and pycrypto (I need to do this to force usage of the IP Input and SimplePairingManager)
- cleaned the entire moonlight cache dir on my Pi
- started Luna, clicked the Add Host button
- entered the IP address
- got prompted with a code, entered it on my Notebook, paired just fine
- entered the game list, waited for the items to appear

The underlying issue here was that I forgot to change the source for the host IP. It can't be obtained from the ConfigHelper anymore, but from the RequestService instead.
I didn't realize that this wasn't working before because I forgot to uninstall pycrypto as well when I tested RC3, so I was still using my own pairing implementation instead of the one you're using.

@knives49 
Sorry for the trouble and thanks for letting me know that quick!
